gp-includes/things/translation.php

```diff
@@ -131,7 +131,7 @@ function for_translation( $project, $translation_set, $page, $filters = array(),
 			$where[] = $wpdb->prepare( 't.user_id = %d', ($user && $user->ID) ? $user->ID : -1 );
 		}
 
-		if ( ! current_user_can( 'manage_options' ) ) {
+		if ( ! GP::$permission->current_user_can( 'admin' ) ) {
```

Comment:
I was curious about it as well, it looks like it let's the "admin" see hidden priority items.

Should be switched back to project/write I would think.